source/gui/__init__.py

```diff
@@ -480,11 +480,32 @@ def onInstallCommand(self, evt):
 
 		installerGui.showInstallGui()
 
+	CRFT_INTRO_MSG: str = _(
+		# Translators: Explain the COM Registration Fixing tool to users before running
+		"Welcome to the COM Registration Fixing tool.\n"
+		"This tool is used by NVDA to fix problems it may have as it tries to interact with various applications, "
+		"or with Windows itself.\n"
+		"It examines the system registry for corrupted or missing accessibility entries and will correct them.\n"
+		"Those entries are sometimes damaged by installing or uninstalling programs, or other system events. This "
+		'can result in "unknown" or "pane" being spoken instead of the content you were expecting, or previously '
+		"accessible elements suddenly no longer reading correctly.\n\n"
+		"You have most likely been asked to run this tool by an experienced user or support worker to investigate "
+		"and potentially fix an issue with NVDA's behaviour.\n"
+		"Because it needs to modify the Windows registry, if you have User Account Control (UAC) active, you will "
+		"be prompted by UAC before this tool can do its job. This is normal and you should follow the UAC's "
+		"instructions to approve the running of the tool.\n\n"
+		"Press Continue to try to repair the registry now.\n",
```

Comment:
Why is there a line feed after the last line of this text?
```suggestion
		"Press Continue to try to repair the registry now.",
```

Reply:
@SaschaCowley I believe that is a remnant from when WX was causing the text to be mingled with the buttons for some reason in a previous implementation. I forgot that it wasn't needed with this one.